sympy/matrices/tests/test_matrices.py

```diff
@@ -2007,7 +2007,7 @@ def test_diff_by_matrix():
 def test_getattr():
```

Comment:
This whole test seems to be based on Matrix having `__getattr__`, which it does not (there's also a comment to that effect in sympy/matrices/common.py). My guess is it used to but it was removed at some point.